### PR TITLE
Replaced AssemblyAI with Whisper

### PR DESCRIPTION
**Pull request by SpecialJayy**
Replaced old AssemblyAi speech to text system with OpenAi's Whisper under the same api key as integration with ChatGPT.

### CODE REVIEW

**Comment by JKGplay** [comment]
test

**Comment by SpecialJayy** [comment]
siema

File: STT.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/hegedustibor/htgo-tts"
 	"github.com/hegedustibor/htgo-tts/handlers"
 	"github.com/joho/godotenv"
+	openai "github.com/sashabaranov/go-openai"
```
Comment (JKGplay) [inline STT.go:18]: git

**Comment by gregklka** [comment]
Hejka, masz konflikt z glownym branchem. Musisz go rozwiązać, bo nie zrobisz merge.

**Pull request closed without merging**